src/app/clusters/power-source-configuration-server/power-source-configuration-server.cpp

```diff
@@ -61,9 +63,16 @@ CHIP_ERROR PowerSourceConfigurationAttrAccess::Read(const ConcreteReadAttributeP
     {
     case Sources::Id:
         err = aEncoder.EncodeList([](const auto & encoder) -> CHIP_ERROR {
+            std::multimap<uint8_t, uint16_t> orderEpMap;
```

Comment:
This is going to do a bunch of heap-allocation, without even using the Matter heap allocator.

And on out-of-heap this will abort, since we are compiling with exceptions disabled....

We need a solution here that is more appropriate to constrained devices.

Reply:
Fixed